Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.9.0-alpine3.12
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+
+COPY ./requirements.txt ./requirements.txt
+
+RUN pip install -r requirements.txt
+
+COPY ./transformer.py ./transformer.py
+
+WORKDIR /conf
+ENTRYPOINT ["python", "transformer.py",  "--input", "/export.json", "--configuration", "/config.json"]
```

`transformer.py` script is not in the current dir but in `/`:

```suggestion
ENTRYPOINT ["python", "/transformer.py",  "--input", "/export.json", "--configuration", "/config.json"]
```

---

Done :)

---

Ah, I forgot, the `export.json` and `config.json` are now in the `conf` directory, which is the current dir in the container, and not `/`

```Dockerfile
ENTRYPOINT ["python", "/transformer.py",  "--input", "./export.json", "--configuration", "./config.json"]
```

Else, you get this error:

```
Traceback (most recent call last):
  File "/transformer.py", line 299, in <module>
    process_conference(parse_json(args.input), parse_json(args.configuration))
  File "/transformer.py", line 281, in parse_json
    with open(path, encoding='UTF-8') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/export.json'
```

---

Done :)